### ISSUE TITLE
CCLabelTTF setColor does not work after progressWithSprite

### ISSUE THREAD

**Issue by hikun** [cocos2d/cocos2d-objc#826]
```
CCLabelTTF *label = [CCLabelTTF labelWithString:@"通" fontName:@"Res/Font/FZXKJW.ttf" fontSize:21];
[label setColor:[CCColor colorWithCcColor3b:ccc3SBallText]];

CCProgressNode *progress = [CCProgressNode progressWithSprite:label];
progress.type = CCProgressNodeTypeBar;
progress.barChangeRate = ccp(0,1);
progress.midpoint = ccp(0.5,0);
progress.percentage = 50.0f;
[progress setPosition:ccp(self.contentSize.width/2, self.contentSize.height/2)];
[self addChild:progress];
```

**Comment by hikun** [comment]
Find this bug in v3.0.0

the setColor works in v3.1.0, but  the word flip horizontal now.

**Comment by slembcke** [comment]
Is it horizontally flipped in the 3.1.0 (not beta) version? The horizontal flipping was fixed right before release.

Somebody submitted a fix for the color property of progress nodes... Maybe it only got merged into 3.1. I'll have to look.
